src/tasks/WasmAppBuilder/WasmAppBuilder.csproj

```diff
@@ -24,13 +24,14 @@
   </ItemGroup>
 
   <Target Name="PublishBuilder"
-          AfterTargets="Build">
+          AfterTargets="Build"
+          Condition="'$(_RunningForPublishBuilder)' != 'true'">
 
     <!-- needed for publishing with multi-targeting. We are publishing essentially to get the SR.MetadataLoadContext.dll :/ -->
     <ItemGroup>
       <_PublishFramework Include="$(TargetFrameworks)" />
     </ItemGroup>
-    <MSBuild Projects="$(MSBuildProjectFile)" Targets="Publish" Properties="TargetFramework=%(_PublishFramework.Identity)" />
+    <MSBuild Projects="$(MSBuildProjectFile)" Targets="Publish" Properties="TargetFramework=%(_PublishFramework.Identity);_RunningForPublishBuilder=true" />
```

Comment:
Global properties added to MSBuild invocation. This is a recipe for a build race condition. The global property defeats the MSBuild results cache and also can cause concurrent builds to the same path if the global property doesn’t change output directories.